### PR TITLE
Removes hardcoded scala version from sbt plugin

### PR DESCRIPTION
**Pull request by octonato**
The hardcoded Scala version is causing cross resolution conflicts for projects using Scala 2.12.

### CODE REVIEW

File: plugin/src/main/scala/com/lightbend/markdown/sbt/LightbendMarkdown.scala [hunk 1]
```diff
@@ -141,13 +141,13 @@ object LightbendMarkdown extends AutoPlugin {
...
-      }
-      "com.lightbend.markdown" % artifact % version % RunMarkdown.name
-
+      val artifact =
+        if (markdownUseBuiltinTheme.value) {
+          "lightbend-markdown-theme-builtin"
+        } else {
+          "lightbend-markdown-server"
+        }
+      "com.lightbend.markdown" % s"${artifact}_${scalaBinaryVersion.value}" % version % RunMarkdown.name
```
Comment (TimMoore) [inline plugin/src/main/scala/com/lightbend/markdown/sbt/LightbendMarkdown.scala]: Does `%%` not work for this?

Comment (octonato) [reply plugin/src/main/scala/com/lightbend/markdown/sbt/LightbendMarkdown.scala]: Absolutely. Dum me!

**Review by TimMoore — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 target
+.idea
```
Comment (dwijnand) [inline .gitignore:2]: Btw you best save yourself the hassle and ignore this globally instead fo repo-by-repo.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Comment (octonato) [reply .gitignore:2]: @dwijnand You are right. Actually, I do it often for other files like `.zshrc` that I usually have per project. Somehow, I keed adding `.idea` to `.gitignore`. 

That will be the last time. ;-)

**Pull request merged**